src/SvgArrow.js

```diff
@@ -181,6 +181,14 @@ const SvgArrow = ({
   const xStart = startingPoint.x;
   const yStart = startingPoint.y;
 
+  /**
+   * If the target is not in the view, endingPoint.x and endingPoint.y are zero.
+   * In this case, do not render the SvgArrow.
+   */
+  if (endingPoint.x === 0 && endingPoint.y === 0) {
```

Comment:
Is there really no other way to check that the element is out of the view?
I wonder if there may be cases where we really want to draw an arrow to `(0, 0)` :thinking:

Reply:
Hmm, thoughts? 

The reason I did this was because on https://github.com/pierpo/react-archer/blob/master/src/ArcherContainer.js#L194

Point(0,0) is returned.

Reply:
Haha, I've done that a while ago!
I think handling the error as I did returning `(0, 0)` was a mistake :thinking: 
I'll think of better ways to do this in the future.

Still, your changes total sense given the situation. I think the case where we draw to `(0, 0)` is very unlikely to happen. But I still need to check this a little bit.

Also, what about checking `startingPoint.x === 0`?

Good job about the PR otherwise. I appreciate the help :blush: . Love the fact that you added a test for your case.

Reply:
Okay, what about returning `Point(NaN, NaN)` in case of error, then checking:

```javascript
  if (
    isNaN(startingPoint.x) ||
    isNaN(startingPoint.y) ||
    isNaN(endingPoint.x) ||
    isNaN(endingPoint.y)
  ) {
    return null;
  }
```

I just tried it out, E2E tests are still green (2 snapshots fail).

Reply:
Question: When would startingPoint be zero/NaN? If the element is not in the view, it would not have relations, so no arrows would be drawn. That's how it behaves. 

We could do Point(NaN, NaN) and then checking for it. 

Re. snapshots, yes they fail but I think they arent related? are they?

Reply:
> Question: When would startingPoint be zero/NaN? If the element is not in the view, it would not have relations, so no arrows would be drawn. That's how it behaves.

Well right now, it draws an arrow that doesn't make any sense to the top left corner :sweat_smile: 

> We could do Point(NaN, NaN) and then checking for it.

Yes that would work. And never draw any arrow in case of error.

> Re. snapshots, yes they fail but I think they arent related? are they?

Haha sorry for the confusion, I just meant that it would be something else to fix. I haven't checked the error precisely. It could be a simple update of the snapshot, or it could be an actual problem that needs thoughts. I don't know yet :smile:

Reply:
@pierpo I just created another PR. I added origin and target as NaN if they are not in the view and also checked for it. 

also I saw the snapshot, I guess they are related to what I did, but I dont know how to fix them.